components/Form/__tests__/MultiStepForm.test.js

```diff
@@ -1,25 +1,25 @@
 /* eslint-disable react/no-multi-comp */
+/* eslint-disable react/sort-comp */
 /* eslint-disable max-classes-per-file */
 import React from 'react';
 import faker from 'faker';
 import get from 'lodash/get';
-import { mount } from 'enzyme'; // eslint-disable-line no-restricted-imports
+import { render, fireEvent, wait } from '@testing-library/react';
 import { Field } from 'formik';
 import * as Yup from 'yup';
 import { networkErrorMessages } from 'common/constants/messages';
-import asyncRenderDiff from 'test-utils/asyncRenderDiff';
 import { MultiStepForm } from '../MultiStepForm';
 
-const submitForm = async enzymeWrapper => {
-  const submitButton = enzymeWrapper.find('button[type="submit"]');
-  submitButton.simulate('submit');
-  await asyncRenderDiff(enzymeWrapper);
+const submitForm = async container => {
+  const button = container.querySelector('button[type="submit"]');
+  fireEvent.submit(button);
+  await wait();
```

Comment:
Can you just wrap the event in the `wait`'s callback?

Reply:
sure